modxcms/revolution
# Add new checkbox TV input option to display inputs as switches/toggles

### Why is it needed?
For some TVs, the switch style may be more appropriate to the content being collected. This update allows users to choose how to display their checkbox TVs.

### How to test
Create a new checkbox TV (or use an existing one) and verify that changing the "Display as Switch" option has the expected effect when the TV is displayed in the Resource editing form.

### Related issue(s)/PR(s)
Alternate solution to PR #16623
